content/en/deployment/strategies/blue-green.md

```diff
@@ -3,78 +3,87 @@ title: "Blue/Green Deployment Strategy"
-
-1. In your deploy file, go to the `strategies` section.
-1. Create a new strategy named `blue-green-deploy-strat` like the following:
-
-   ```yaml
-   strategies:
-    blue-green-deploy-strat:
-      blueGreen:
-        redirectTrafficAfter:
-          - pause:
-              untilApproved: true
-        shutDownOldVersionAfter:
-          - pause:
-              untilApproved: true
-   ```
-
-   See the [Deployment File Reference]({{< ref "reference/deployment/config-file/strategies#bluegreen-fields" >}}) for an explanation of these fields.
-
-   This strategy is configured to pause for manual judgment before redirecting traffic to your new app version as well as before shutting down your old version. You could instead choose to pause for a duration of time.
-
-1. Change the value of `targets.<targetName>.strategy` for one or more of your deployment targets to `blue-green-deploy-strat`.
-
-   ```yaml
-   ...
-   targets:
-    <targetName>:
-      account: <agentIdentifier>
-      namespace: <namespace>
-      strategy: blue-green-deploy-strat
-    ...
-    ```
-1. At the bottom of your file, create a top-level traffic management configuration for your `activeService` and `previewService`:
-
-   ```yaml
-   ...
-   trafficManagement:
-     - targets: ['<targetName>']
-       kubernetes:
-         - activeService: myAppActiveService
-           previewService: myAppPreviewService
-   ...
-   ```
+For this guide, you need the following:
+
+- The [Armory CLI]({{< ref "cli.md" >}}) installed
+- A [Remote Network Agent]({{< ref "remote-network-agent/overview.md" >}}) (RNA) installed in the cluster where you want to deploy your app
+
+
+## Deploy your active service 
+You need to deploy a [Kubernetes Service object](https://kubernetes.io/docs/concepts/services-networking/service/) that sends traffic to the current version of your application. This is the `trafficManagement.kubernetes.activeService` field in the YAML configuration.
+
+As an example:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: <your-app-service>
+  labels:
+    app: <your-app-name>
+spec:
+  selector:
+    app: <your-app-name>
+  ports:
+    - name: http
+      port: 80
+      targetPort: 9001
+      protocol: TCP
+```
+
+Save the file as `<your-app-service>.yaml` and run `kubectl -n <target-namespace> -f <your-app-service>.yaml` to deploy the service.
+
+## Create a blue/green deployment file
+
+Create a `blue-green-deployment.yaml` file with the following contents:
+
+```yaml
+version: v1
+kind: kubernetes
+application: <your-app-name> # the name of your app
+targets:
+  staging:  
+    account: <your-remote-network-agent-identifier> # the name of the RNA you installed in your cluster
+    namespace: <your-namespace> # defined in namespace-staging.yaml
+    strategy: blue-green-deploy-strat
+manifests:
+  - path: manifests/your-app.yaml # replace with the name of your app manifest
+strategies:
+  blue-green-deploy-strat:
+    blueGreen:
+      redirectTrafficAfter:
```

Comment:
We need to add more text information for the steps here`redirectTrafficAfter` and `shutDownOldVersionAfter`. Doesn't have to be in the yaml file